package.json

```diff
@@ -24,6 +24,8 @@
   "dependencies": {
     "bcrypt": "^1.0.3",
     "body-parser": "^1.18.2",
+    "chai": "^4.1.2",
+    "chai-http": "git+https://github.com/chaijs/chai-http.git",
```

Comment:
Why use `chai-http`? I think `chai-http` is a descendant already of supertest?

Reply:
I don't know that, maybe because using chai-http centralized the plugins that we are using, cause we already use chai on assertion

Reply:
I see, any pros and cons of using it?

Reply:
Not that much pros, aside on centralized package but cons of it I think none